dali/CMakeLists.txt

```diff
@@ -92,6 +92,10 @@ add_custom_command(
 # Build test suite
 ################################################
 if (BUILD_TEST)
+  # define directory for test plugins
+  set(TEST_PLUGINS_BINARY_DIR ${PROJECT_BINARY_DIR}/dali/test/plugins)
+  add_definitions( "-DTEST_PLUGINS_BINARY_DIR=\"${TEST_PLUGINS_BINARY_DIR}\"" )
```

Comment:
This is fine for source distributions; not so good for binary ones (this directory will point to some CMake-generated directory)